### PR TITLE
fix: add posts pages to use the new feed layout

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes
- added pages to the set we use to validate

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [x] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [x] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-394 #done

### Preview domain
https://mi-394-posts-layout.preview.app.daily.dev

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]

File: packages/shared/src/hooks/useFeedLayout.ts [hunk 1]
```diff
@@ -58,6 +58,10 @@ export const FeedLayoutMobileFeedPages = new Set([
   OtherFeedPage.SearchBookmarks,
   OtherFeedPage.UserUpvoted,
   OtherFeedPage.UserPosts,
+  OtherFeedPage.Explore,
+  OtherFeedPage.ExploreLatest,
+  OtherFeedPage.ExploreDiscussed,
+  OtherFeedPage.ExploreUpvoted,
```
Comment (sshanzel) [inline packages/shared/src/hooks/useFeedLayout.ts:64]: Asking again here. Why did we end up moving these feed pages to `OtherFeedPage`. The reason we had it at `SharedFeedPage` is to understand that they are shared across both webapp and extension.

https://github.com/dailydotdev/apps/pull/3237#discussion_r1645933777

**Comment by sshanzel** [comment]
Still getting this and some layout shift issues.

Screenshot 2024-06-21 at 4 43 14 PM

**Comment by sshanzel** [comment]
https://github.com/dailydotdev/apps/assets/13744167/d8fb6893-7651-4ac9-af82-3d979c2c51a9

**Review by sshanzel — CHANGES_REQUESTED** [review]
Kindly check the screenshots and recording mentioned.

File: packages/shared/src/hooks/useFeedLayout.ts (hunk 1 above)
Comment (nensidosari) [reply packages/shared/src/hooks/useFeedLayout.ts:64]: BTW I didn't know SharedFeedPage had that purpose as well to differentiate for what is shared in the extension.

So SharedFeedPages are what is checked for wether to redirect to /onboarding, those pages are what is behind auth. 

If we want to keep them at SharedFeedPages then we just need to filter the /posts out when checking for authenticaiton

**Comment by nensidosari** [comment]
> Also, all posts pages are now broken with padding.

@sshanzel this is what I keep mentioning, it's not broken, it's an accepted issue, happens only on mobile when u navigate with the link, so u use ssr page. If u navigate through buttons within the page then u never get these paddings.

**Comment by nensidosari** [comment]
@sshanzel as for the video, if for desktop now it has more paddings that it did before I can check and place the right ones there.

**Comment by sshanzel** [comment]
> @sshanzel as for the video, if for desktop now it has more paddings that it did before I can check and place the right ones there.

The concern was more of around this content and the shift that it makes. Or at least what it looks like it makes.
Screenshot 2024-06-21 at 4 55 47 PM

**Comment by sshanzel** [comment]
> > Also, all posts pages are now broken with padding.
> 
> @sshanzel this is what I keep mentioning, it's not broken, it's an accepted issue, happens only on mobile when u navigate with the link, so u use ssr page. If u navigate through buttons within the page then u never get these paddings.

Oh, wow, the UI looks quite bad. Well, if it is an accepted minor issue, then cool, let's move forward with it. Even with SSR, we should be able to force a re-render at this point that's why I didn't understand how it relates to SSR at first.

**Comment by sshanzel** [comment]
A few more pages that broke.

Screenshot 2024-06-21 at 5 01 56 PM
Screenshot 2024-06-21 at 5 02 02 PM

**Comment by nensidosari** [comment]
> The concern was more of around this content and the shift that it makes. Or at least what it looks like it makes.

@sshanzel that's already an issue on prod

**Comment by nensidosari** [comment]
> A few more pages that broke.

@sshanzel I don't know how u get this pages, whats the url? maybe u're in prod?

**Comment by sshanzel** [comment]
> > The concern was more of around this content and the shift that it makes. Or at least what it looks like it makes.
> 
> @sshanzel that's already an issue on prod

Oh, I don't seem to get it. But cool. If it is a known issue as long as we have a ticket for it already.

https://github.com/dailydotdev/apps/assets/13744167/ca310164-5ab4-43ba-85dc-ea1f64e63e5c

**Review by sshanzel — APPROVED** [review]
Looks like more of my concerns are already a known issue, so approving.

File: packages/shared/src/hooks/useFeedLayout.ts (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/hooks/useFeedLayout.ts:64]: No they are SEO pages so highly important they are shown to bots and not redirect 🙏

**Comment by nensidosari** [comment]
> Oh, I don't seem to get it. But cool. If it is a known issue as long as we have a ticket for it already.

U'll get it if u play with throttle @sshanzel

**Comment by rebelchris** [comment]
https://github.com/dailydotdev/apps/assets/554874/b9298610-1062-4bf5-b4ea-b2807a6c02e3

Theres a weird difference in padding for 2/4 feeds

**Comment by nensidosari** [comment]
So end result all issues mentioned above are issues happening in prod as well, so moving on with merging this.

File: packages/shared/src/hooks/useFeedLayout.ts (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/hooks/useFeedLayout.ts:64]: Oh. So we are back to allowing feeds to anonymous users now?

Comment (rebelchris) [reply packages/shared/src/hooks/useFeedLayout.ts:64]: @sshanzel Not all, just the explore ones as they are for SEO purposes mainly (addition people can visit them yeah)
(Same for tag page where we removed the auth banner as well for instance)

Comment (nensidosari) [reply packages/shared/src/hooks/useFeedLayout.ts:64]: Sorry for not getting back to u on this Lee, see u already asked product :D

**Pull request merged**